### ISSUE TITLE
qgroundcontrol checksum mismatch

### ISSUE THREAD

**Issue by jpkotta** [acxz/pkgbuilds#104]
```
==> Validating source files with sha256sums...
    qgroundcontrol-4.1.0.tar.gz ... FAILED
```

```
$ sha256sum qgroundcontrol-4.1.0.tar.gz
06ef7e98a4adc06750e6f8a0fcd12eef011d8c93aedc55b9ca45d3fb637b9d24  qgroundcontrol-4.1.0.tar.gz
```

Seems like upstream edited the release 3 days ago.

**Comment by acxz** [comment]
rip feels bad, let me look at this rq

**Comment by acxz** [comment]
should be fixed with 691f8d3
lmk if you have any other issues
thx for the report
